.github/CODEOWNERS

```
@@ -40,7 +40,8 @@
 /StRoot/StDetectorDbMaker/St_vertexSeed*           @genevb
 /StRoot/StEEmc*                                    @kkauder           @rkunnawa
 /StRoot/StEEmcSimulatorMaker                       @kkauder           @rkunnawa    @klendathu2k
-/StRoot/StEEmcUtil                                 @kkauder           @rkunnawa    @klendathu2k
+/StRoot/StEEmcUtil                                 @kkauder           @rkunnawa    @klendathu2k @tinglin-physics
+/StRoot/StEEmcPool                                 @tinglin-physics
```

```suggestion
/StRoot/StEEmcPool                                 @kkauder           @rkunnawa    @tinglin-physics
```
To not remove others from codeowners

---

It might be a good time to reconsider owners for the EMC/EEMC code. According to this page https://drupal.star.bnl.gov/STAR/comp/org @kkauder is not a software coordinator for these subsystems